### PR TITLE
authy: 2.1.0 -> 2.2.0

### PR DESCRIPTION
**Pull request by SFrijters**
###### Description of changes

A new version is available and it doesn't seem to require the old (insecure) `electron_9` package anymore.
Also, the resulting output is rather big and it seems that some stuff is not actually needed?
Maybe instead of copying everything and deleting some stuff there should just be a minimal list of items to be copied to `$out`.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SFrijters** [comment]
Result of `nixpkgs-review pr 175206` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>authy</li>
  </ul>
</details>

File: pkgs/applications/misc/authy/default.nix [hunk 1]
```diff
@@ -6,7 +6,7 @@
 , cairo
 , cups
 , dbus
-, electron_9
+, electron
```
Comment (davegallant) [inline pkgs/applications/misc/authy/default.nix:9]: :+1:

**Review by davegallant — APPROVED** [review]
Works on my machine.

**Review by iAmMrinal0 — APPROVED** [review]

**Comment by iAmMrinal0** [comment]
Package builds, works, and is the version it's updated to.

**Comment by SFrijters** [comment]
I tried to clean things up some more, by copying only the minimum amount of stuff needed, and it still works for me.

Before
```
/nix/store/v9fl3zmr0j26qj5id93vf6y4n9gfp3sd-authy-2.2.0 $ du -h
5.9M	./swiftshader
8.0K	./bin
8.4M	./resources
8.0M	./locales
648K	./lib/x86_64-linux-gnu
652K	./lib
4.0K	./share/pixmaps/apps
52K	./share/pixmaps
8.0K	./share/applications
64K	./share
39M	.
```

After:

```
/nix/store/3dlpxj10ysaq86vkm02hhg9d89kq5fhd-authy-2.2.0 $ du -h
8.0K	./bin
8.4M	./resources
8.0M	./locales
4.0K	./share/pixmaps/apps
52K	./share/pixmaps
8.0K	./share/applications
64K	./share
22M	.
```

**Comment by SFrijters** [comment]
Result of `nixpkgs-review pr 175206` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>authy</li>
  </ul>
</details>

**Comment by SFrijters** [comment]
I'm using this now via an overlay and I haven't had any problems so far.

**Pull request merged**